cognite/client/data_classes/simulators/logs.py

```diff
@@ -0,0 +1,58 @@
+from __future__ import annotations
+
+from dataclasses import dataclass
+from typing import TYPE_CHECKING, Any, Literal
+
+from typing_extensions import Self
+
+from cognite.client.data_classes._base import (
+    CogniteObject,
+    CogniteResource,
+    CogniteResourceList,
+    IdTransformerMixin,
+)
+
+if TYPE_CHECKING:
+    from cognite.client import CogniteClient
+
+SEVERITY = Literal["Debug", "Information", "Warning", "Error"]
```

Comment:
Type aliases should only have their first letter capitalized
```suggestion
Severity = Literal["Debug", "Information", "Warning", "Error"]
```